tests/testthat/test-size.R

```diff
@@ -49,6 +49,15 @@ test_that("can subset object of any dimensionality", {
   expect_equal(vec_slice(x4, 1L), ones(1, 3, 4, 5))
 })
 
+test_that("can subset data frame columns", {
+  df <- data.frame(x = c(1, 2), y = I(data.frame(z = c(1, 2), w = c(1, 2))))
```

Comment:
I think it's slightly better to add after the fact:

```R
df <- data.frame(x = 1:2)
df$y <- data.frame(a = 2:1)
```

Reply:
That makes more sense, especially considering I couldn't even print `df` before